src/main/java/codecrafter47/bungeemail/BungeeMail.java

```diff
@@ -201,19 +209,21 @@ public void sendMailToAll(ProxiedPlayer sender, String text) {
             sender.sendMessage(getChatParser().parse(config.getString("commandError", "&cAn error occurred while processing your command: %error%").replace("%error%", e.getMessage())));
             return;
         }
+        targets.add(CONSOLE_UUID);
         text = BBCodeChatParser.stripBBCode(text);
         text = text.replaceAll("(?<link>(?:(https?)://)?([-\\w_\\.]{2,}\\.[a-z]{2,4})(/\\S*)?)", "[url]${link}[/url]");
         for (UUID targetUUID : targets) {
-            if (targetUUID.equals(sender.getUniqueId())) continue;
+            if (targetUUID.equals(senderUUID)) continue;
             try {
-                storage.saveMessage(sender.getName(), sender.getUniqueId(), targetUUID, text, false, time);
+                storage.saveMessage(sender.getName(), senderUUID, targetUUID, text, false, time);
                 if (getProxy().getPlayer(targetUUID) != null) {
                     getProxy().getPlayer(targetUUID).sendMessage(chatParser.parse(config.getString("receivedNewMessage")));
                 }
             } catch (StorageException e) {
                 getLogger().log(Level.WARNING, "Unable to save mail", e);
             }
         }
+        getProxy().getConsole().sendMessage(chatParser.parse(config.getString("receivedNewMessage")));
```

Comment:
Only if console is not the source, I think?

Reply:
Yeah, you are right.